workshop/Lab4/README.md

```diff
@@ -0,0 +1,90 @@
+# *** UNDER CONSTRUCTION ***
+
+# 3. Check the health of apps
```

Comment:
I know its under construction, but let's make this 1 instead of 3

Reply:
faxed